spark/src/test/scala/io/delta/tables/DeltaTableSuite.scala

```diff
@@ -234,6 +234,7 @@ class DeltaTableHadoopOptionsSuite extends QueryTest
   private val publicMethods =
   scala.reflect.runtime.universe.typeTag[io.delta.tables.DeltaTable].tpe.decls
     .filter(_.isPublic)
+    .filter(!_.isSynthetic)
```

Comment:
@Kimahriman If we remove the default values for `DeltaTable`'s Scala APIs we would no longer need to add this `isSynthetic` filter here - https://github.com/delta-io/delta/pull/3392/files#r1720202542

Reply:
Removed